core/utils.py

```diff
@@ -264,3 +264,15 @@ async def update_presence(bot: commands.Bot, *, status: str = None) -> None:
                 name=config.prefix + "help",
             ),
         )
+
+    @staticmethod
+    async def send_dm(
+        user: Union[discord.Member, discord.User],
+        text: Optional[str] = None,
+        embed: Optional[discord.Embed] = None,
+    ) -> bool:
+        try:
```

Comment:
When both text and embed are empty, this would throw an error. We should raise `ValueError` if both are `None`.

Reply:
Is raising a ValueError with a message like "Could not send an empty message." good enough?

Reply:
Yeah, sure, something like that.